build.sh

```diff
@@ -0,0 +1,179 @@
+#! /bin/bash
```

Comment:
I would love to replace our powershell script with this, much rather maintain a single platform agnostic one vs having two different ones for windows and *nix.